web/modules/custom/campuschampions/campuschampions.install

```diff
@@ -549,3 +549,32 @@ function campuschampions_update_9000() {
     ]
   ]);
 }
+
+/**
+ * Add LAT and LON fields to carnegie_codes table
+ */
+function campuschampions_update_9001() {
+  $table = 'carnegie_codes';
+  $schema = \Drupal::database()->schema();
+
+  if ($schema->tableExists($table)) {
+    if (!$schema->fieldExists($table, 'LAT')) {
+      $schema->addField($table, 'LAT', array(
+        'type' => 'decimal',
```

Comment:
Looks like we need to use numeric type instead of decimal in Drupal Schema: https://www.drupal.org/docs/7/api/schema-api/data-types/decimal-numeric

It would be nice to use the

Reply:
Fixed. I completely missed that (obviously). Thanks.